File: Latihan1b/Java/TimSepakBola.java
public class TimSepakBola
{
    private String nama;
    private String asalNegara;
    private String tahunBerdiri;
    private String pemain;

    // constructor
    public TimSepakBola() {}

    public TimSepakBola(String nama, String asalNegara, String tahunBerdiri, String pemain)
    {
        this.nama = nama;
        this.asalNegara = asalNegara;
        this.tahunBerdiri = tahunBerdiri;
        this.pemain = pemain;
    }

    // setter and getter method
    public void setNama(String nama)
    {
        this.nama = nama;
    }

    public String getNama()
    {
        return this.nama;
    }

    public void setAsalNegara(String asalNegara)
    {
        this.asalNegara = asalNegara;
    }

    public String getAsalNegara()
    {
        return this.asalNegara;
    }

    public void setTahunBerdiri(String tahunBerdiri)
    {
        this.tahunBerdiri = tahunBerdiri;
    }

    public String getTahunBerdiri()
    {
        return this.tahunBerdiri;
    }

    public void setPemain(String pemain)
    {
        this.pemain = pemain;
    }

    public String getPemain()
    {
        return this.pemain;
    }

    // output
    public void printOut()
    {
        System.out.println("Nama Tim        : " + this.getNama());
        System.out.println("Asal Negara     : " + this.getAsalNegara());
        System.out.println("Tahun Berdiri   : " + this.getTahunBerdiri());
        System.out.println("Pemain          : " + this.getPemain());
    }
}

File: Latihan1b/Python/TimSepakBola.py
class TimSepakBola():
    __nama = ""
    __asalNegara = ""
    __tahunBerdiri = ""
    __pemain = ""

    # constructor
    def __init__(self, nama = "", asalNegara = "", tahunBerdiri = "", pemain  = ""):
        self.__nama = nama
        self.__asalNegara = asalNegara
        self.__tahunBerdiri = tahunBerdiri
        self.__pemain = pemain

    # setter and getter method
    def setNama(self, nama):
        self.__nama = nama

    def getNama(self):
        return self.__nama
    
    def setAsalNegara(self, asalNegara):
        self.__asalNegara = asalNegara
    
    def getAsalNegara(self):
        return self.__asalNegara

    def setTahunBerdiri(self, tahunBerdiri):
        self.__tahunBerdiri = tahunBerdiri
    
    def getTahunBerdiri(self):
        return self.__tahunBerdiri

    def setPemain(self, pemain):
        self.__pemain = pemain
    
    def getPemain(self):
        return self.__pemain

    # output
    def printOut(self):
        print("Nama Tim         : " + str(self.__nama))
        print("Asal Negara      : " + str(self.__asalNegara))
        print("Tahun Berdiri    : " + str(self.__tahunBerdiri))
        print("Pemain           : " + str(self.__pemain))